server/tests/backends/fixtures/filter/dates_from_until_pypika.json

```diff
@@ -0,0 +1,63 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "filter",
+        "condition": {
+          "and": [
+            {
+              "column": "brewing_date",
+              "operator": "from",
+              "value": "2022-01-07 00:00:00"
+            },
+            {
+              "column": "brewing_date",
+              "operator": "until",
+              "value": "2022-01-05 00:00:00"
+            }
+          ]
```

Comment:
looks like this logic was not good since then here : https://github.com/ToucanToco/weaverbird/blob/master/server/src/weaverbird/backends/pypika_translator/translators/base.py#L520

thanks for catching this up !

i updated from your remark here : https://github.com/ToucanToco/weaverbird/pull/1413/commits/9acc35649efc5a65680a5554db522703903d69ae

Let me know what you think !